public/src/components/channelManagement/bannerTests/bannerVariantPreview.tsx

```diff
@@ -152,6 +152,10 @@ const bannerModules = {
     path: 'postElectionAuMoment/PostElectionAuMomentMorrisonBanner.js',
     name: 'PostElectionAuMomentMorrisonBanner',
   },
+  [BannerTemplate.AusBrandMomentBanner]: {
+    path: 'ausBrandMoment/ausBrandMomentBanner.js',
```

Comment:
This just needs to be `ausBrandMoment/AusBrandMomentBanner.js`

Reply:
Thanks!